infrastructure/prod.tfvars

```diff
@@ -0,0 +1 @@
+capacity = "2"
```

Comment:
Is this the number of app servers we'll have? Perhaps we could just have 1, it'd make writing the ingestion cron easier.

Reply:
I wouldn't worry for now as Azure Storage has locking mechanism built in. We can change it later.